DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: epiprocess
 Title: Tools for basic signal processing in epidemiology
-Version: 0.9.0
+Version: 0.10.1
```

Comment:
I think we should be in the 0.9.x territory, since we just incremented to 0.9.0 from 0.8.x in the slide improvements.
```suggestion
Version: 0.9.1
```

Reply:
Should changelog notes still be under 0.10?

Reply:
Hm, my thinking was we append to 0.x whenever we bump 0.x.n to 0.x.n+1 and when we make a transition to 0.x+1, we write the changes that went along with that change under 0.x+1. So here we'd just be appending to the list of changes in 0.9.

Reply:
I commented on other PR about this but I'm just going to temporarily match 0.7.y and 0.8.y, y>0 behavior here of adding changes under next 0.x heading.